### ISSUE TITLE
Possible crash in GC

### ISSUE THREAD

**Issue by jensjoha** [dart-lang/sdk#40157]
Recently there's been a few crashes on the flutter-frontend bot:

https://logs.chromium.org/logs/dart/buildbucket/cr-buildbucket.appspot.com/8891539827416900384/+/steps/compile_flutter_tests/0/stdout?format=raw (https://ci.chromium.org/p/dart/builders/ci.sandbox/flutter-frontend/3484)
https://logs.chromium.org/logs/dart/buildbucket/cr-buildbucket.appspot.com/8891529813880713136/+/steps/compile_flutter_tests/0/stdout?format=raw (https://ci.chromium.org/p/dart/builders/ci.sandbox/flutter-frontend/3487)
https://logs.chromium.org/logs/dart/buildbucket/cr-buildbucket.appspot.com/8891403841234519136/+/steps/compile_flutter_tests/0/stdout?format=raw (https://ci.chromium.org/p/dart/builders/ci.sandbox/flutter-frontend/3503)

They crash something like this --- seemingly in the `Scavenger`:
```
===== CRASH =====
si_signo=Segmentation fault(11), si_code=1, si_addr=0x7f8508fa56e8
version=2.8.0-edge.33d468dcebf8e32e6ee3d90692a8de132fae36f4 (Sat Jan 11 17:30:14 2020 +0000) on "linux_x64"
thread=31275, isolate=main(0x55f6e8ce7800)
  pc 0x000055f6e7ecd9bf fp 0x00007f853cd7b720 /b/s/w/ir/cache/builder/sdk/out/ReleaseX64/dart-sdk/bin/dart+0x1c459bf
  pc 0x000055f6e7ec7aa4 fp 0x00007f853cd7b770 dart::PageSpace::VisitRememberedCards(dart::ObjectPointerVisitor*) const
  pc 0x000055f6e7ecc592 fp 0x00007f853cd7b7c0 dart::Scavenger::IterateStoreBuffers(dart::Isolate*, dart::ScavengerVisitor*)
  pc 0x000055f6e7ecc68a fp 0x00007f853cd7b840 dart::Scavenger::IterateRoots(dart::Isolate*, dart::ScavengerVisitor*)
  pc 0x000055f6e7ecd419 fp 0x00007f853cd7b9b0 dart::Scavenger::Scavenge()
  pc 0x000055f6e7ec0b8b fp 0x00007f853cd7ba70 dart::Heap::CollectNewSpaceGarbage(dart::Thread*, dart::Heap::GCReason)
  pc 0x000055f6e7ebf60f fp 0x00007f853cd7baf0 dart::Heap::AllocateOld(long, dart::HeapPage::PageType)
  pc 0x000055f6e7c572e1 fp 0x00007f853cd7bb40 dart::Object::Allocate(long, long, dart::Heap::Space)
  pc 0x000055f6e7c9f783 fp 0x00007f853cd7bb90 dart::TypedData::New(long, long, dart::Heap::Space)
  pc 0x000055f6e7c50130 fp 0x00007f853cd7bc00 dart::NativeEntry::BootstrapNativeCallWrapper(_Dart_NativeArguments*, void (*)(_Dart_NativeArguments*))
  pc 0x00007f853cd81169 fp 0x00007f853cd7bc40 Unknown symbol
  pc 0x00007f853bf1ec7f fp 0x00007f853cd7bc80 Unknown symbol
  pc 0x00007f8510ecc0c2 fp 0x00007f853cd7bcd0 Unknown symbol
  pc 0x00007f8510ecba58 fp 0x00007f853cd7bd28 Unknown symbol
  pc 0x00007f84fc732707 fp 0x00007f853cd7bd88 Unknown symbol
  pc 0x00007f8510ecf50d fp 0x00007f853cd7bdc8 Unknown symbol
  pc 0x00007f850d209edd fp 0x00007f853cd7be98 Unknown symbol
  pc 0x00007f8510eadc2b fp 0x00007f853cd7bef8 Unknown symbol
  pc 0x00007f8533b2298a fp 0x00007f853cd7bf88 Unknown symbol
  pc 0x00007f8510ead753 fp 0x00007f853cd7bfd0 Unknown symbol
  pc 0x00007f8510ead262 fp 0x00007f853cd7c028 Unknown symbol
  pc 0x00007f8514fc2a4a fp 0x00007f853cd7c110 Unknown symbol
  pc 0x00007f8510e9cdbd fp 0x00007f853cd7c1b8 Unknown symbol
  pc 0x00007f85126db002 fp 0x00007f853cd7c1e0 Unknown symbol
  pc 0x00007f8510e9c298 fp 0x00007f853cd7c288 Unknown symbol
  pc 0x00007f8534235666 fp 0x00007f853cd7c370 Unknown symbol
  pc 0x00007f853425f829 fp 0x00007f853cd7c3d0 Unknown symbol
  pc 0x00007f850fa24563 fp 0x00007f853cd7c440 Unknown symbol
  pc 0x00007f850fa7bef2 fp 0x00007f853cd7c4d0 Unknown symbol
  pc 0x00007f850fa1ca9d fp 0x00007f853cd7c538 Unknown symbol
  pc 0x00007f8515f2a48d fp 0x00007f853cd7c588 Unknown symbol
  pc 0x00007f85256241e7 fp 0x00007f853cd7c5c8 Unknown symbol
  pc 0x00007f8534274227 fp 0x00007f853cd7c660 Unknown symbol
  pc 0x00007f84fc74e673 fp 0x00007f853cd7c6a8 Unknown symbol
  pc 0x00007f853bf4d3f8 fp 0x00007f853cd7c738 Unknown symbol
  pc 0x00007f8515f3b187 fp 0x00007f853cd7c7c8 Unknown symbol
  pc 0x00007f851823da32 fp 0x00007f853cd7c840 Unknown symbol
  pc 0x00007f850fa24541 fp 0x00007f853cd7c8b0 Unknown symbol
  pc 0x00007f850fa7bef2 fp 0x00007f853cd7c940 Unknown symbol
  pc 0x00007f850fa1ca9d fp 0x00007f853cd7c9a8 Unknown symbol
  pc 0x00007f8515f2a48d fp 0x00007f853cd7c9f8 Unknown symbol
  pc 0x00007f85256241e7 fp 0x00007f853cd7ca38 Unknown symbol
  pc 0x00007f850fa096ba fp 0x00007f853cd7cf10 Unknown symbol
  pc 0x00007f84fc74e673 fp 0x00007f853cd7cf58 Unknown symbol
  pc 0x00007f853bf4d3f8 fp 0x00007f853cd7cfe8 Unknown symbol
  pc 0x00007f8515f3b187 fp 0x00007f853cd7d078 Unknown symbol
  pc 0x00007f851823da32 fp 0x00007f853cd7d0f0 Unknown symbol
  pc 0x00007f850fa24541 fp 0x00007f853cd7d160 Unknown symbol
  pc 0x00007f850fa7bef2 fp 0x00007f853cd7d1f0 Unknown symbol
  pc 0x00007f850fa1ca9d fp 0x00007f853cd7d258 Unknown symbol
  pc 0x00007f8515f2a48d fp 0x00007f853cd7d2a8 Unknown symbol
  pc 0x00007f85256241e7 fp 0x00007f853cd7d2e8 Unknown symbol
  pc 0x00007f8533654b10 fp 0x00007f853cd7d388 Unknown symbol
  pc 0x00007f84fc74e673 fp 0x00007f853cd7d3d0 Unknown symbol
  pc 0x00007f853bf4d3f8 fp 0x00007f853cd7d460 Unknown symbol
  pc 0x00007f8515f3b187 fp 0x00007f853cd7d4f0 Unknown symbol
  pc 0x00007f851823da32 fp 0x00007f853cd7d568 Unknown symbol
  pc 0x00007f850fa24541 fp 0x00007f853cd7d5d8 Unknown symbol
  pc 0x00007f850fa7bef2 fp 0x00007f853cd7d668 Unknown symbol
  pc 0x00007f850fa1ca9d fp 0x00007f853cd7d6d0 Unknown symbol
  pc 0x00007f8515f2a48d fp 0x00007f853cd7d720 Unknown symbol
  pc 0x00007f85256241e7 fp 0x00007f853cd7d760 Unknown symbol
  pc 0x00007f851825d4fa fp 0x00007f853cd7d7f8 Unknown symbol
  pc 0x00007f84fc74e673 fp 0x00007f853cd7d840 Unknown symbol
  pc 0x00007f853bf4d3f8 fp 0x00007f853cd7d8d0 Unknown symbol
  pc 0x00007f8515f3b187 fp 0x00007f853cd7d960 Unknown symbol
  pc 0x00007f851823da32 fp 0x00007f853cd7d9d8 Unknown symbol
  pc 0x00007f850fa24541 fp 0x00007f853cd7da48 Unknown symbol
  pc 0x00007f850fa7bef2 fp 0x00007f853cd7dad8 Unknown symbol
  pc 0x00007f850fa1ca9d fp 0x00007f853cd7db40 Unknown symbol
  pc 0x00007f8515f2a48d fp 0x00007f853cd7db90 Unknown symbol
  pc 0x00007f85256241e7 fp 0x00007f853cd7dbd0 Unknown symbol
  pc 0x00007f8514fcaf46 fp 0x00007f853cd7dd20 Unknown symbol
  pc 0x00007f84fc74e673 fp 0x00007f853cd7dd68 Unknown symbol
  pc 0x00007f853bf4d3f8 fp 0x00007f853cd7ddf8 Unknown symbol
  pc 0x00007f8515f3b187 fp 0x00007f853cd7de88 Unknown symbol
  pc 0x00007f851823da32 fp 0x00007f853cd7df00 Unknown symbol
  pc 0x00007f850fa24541 fp 0x00007f853cd7df70 Unknown symbol
  pc 0x00007f850fa7bef2 fp 0x00007f853cd7e000 Unknown symbol
  pc 0x00007f850fa1ca9d fp 0x00007f853cd7e068 Unknown symbol
  pc 0x00007f8515f2a48d fp 0x00007f853cd7e0b8 Unknown symbol
  pc 0x00007f85256241e7 fp 0x00007f853cd7e0f8 Unknown symbol
  pc 0x00007f84fc71b0f8 fp 0x00007f853cd7e2b8 Unknown symbol
  pc 0x00007f84fc74e673 fp 0x00007f853cd7e300 Unknown symbol
  pc 0x00007f853bf4d3f8 fp 0x00007f853cd7e390 Unknown symbol
  pc 0x00007f8515f3b187 fp 0x00007f853cd7e420 Unknown symbol
  pc 0x00007f851823da32 fp 0x00007f853cd7e498 Unknown symbol
  pc 0x00007f850fa24541 fp 0x00007f853cd7e508 Unknown symbol
  pc 0x00007f850fa7bef2 fp 0x00007f853cd7e598 Unknown symbol
  pc 0x00007f850fa1ca9d fp 0x00007f853cd7e600 Unknown symbol
  pc 0x00007f8515f2a48d fp 0x00007f853cd7e650 Unknown symbol
  pc 0x00007f853425c81a fp 0x00007f853cd7e6a0 Unknown symbol
  pc 0x00007f850fa65b69 fp 0x00007f853cd7e738 Unknown symbol
  pc 0x00007f84fc771f1b fp 0x00007f853cd7e7b0 Unknown symbol
  pc 0x00007f8514ffabaf fp 0x00007f853cd7e820 Unknown symbol
  pc 0x00007f84fc7716f4 fp 0x00007f853cd7e860 Unknown symbol
  pc 0x00007f84fc7351ce fp 0x00007f853cd7e890 Unknown symbol
  pc 0x00007f8534226211 fp 0x00007f853cd7e8d0 Unknown symbol
  pc 0x00007f8534226127 fp 0x00007f853cd7e910 Unknown symbol
  pc 0x00007f8534225f26 fp 0x00007f853cd7e950 Unknown symbol
  pc 0x00007f853bf2afdd fp 0x00007f853cd7e978 Unknown symbol
  pc 0x00007f853cd8149c fp 0x00007f853cd7e9e8 Unknown symbol
  pc 0x000055f6e7bec2af fp 0x00007f853cd7ea90 dart::DartEntry::InvokeFunction(dart::Function const&, dart::Array const&, dart::Array const&, unsigned long)
  pc 0x000055f6e7bee28f fp 0x00007f853cd7eae0 dart::DartLibraryCalls::HandleMessage(dart::Object const&, dart::Instance const&)
  pc 0x000055f6e7c23746 fp 0x00007f853cd7ecd0 dart::IsolateMessageHandler::HandleMessage(std::__2::unique_ptr<dart::Message, std::__2::default_delete<dart::Message> >)
  pc 0x000055f6e7c4e290 fp 0x00007f853cd7ed50 dart::MessageHandler::HandleMessages(dart::MonitorLocker*, bool, bool)
  pc 0x000055f6e7c4ec50 fp 0x00007f853cd7edb0 dart::MessageHandler::TaskCallback()
  pc 0x000055f6e7d4ccf7 fp 0x00007f853cd7ede0 dart::ThreadPool::Worker::Loop()
  pc 0x000055f6e7d4cb7e fp 0x00007f853cd7ee10 dart::ThreadPool::Worker::Main(unsigned long)
  pc 0x000055f6e7ccbfb6 fp 0x00007f853cd7eed0 /b/s/w/ir/cache/builder/sdk/out/ReleaseX64/dart-sdk/bin/dart+0x1a43fb6
-- End of DumpStackTrace
[exit     : sp(0) fp(0x7f853cd7bc40) pc(0)]
[dart     : sp(0x7f853cd7bc50) fp(0x7f853cd7bc80) pc(0x7f853bf1ec7f) dart:typed_data_Uint8List_Uint8List. ]
[dart     : sp(0x7f853cd7bc90) fp(0x7f853cd7bcd0) pc(0x7f8510ecc0c2) dart:io__CopyingBytesBuilder@15069316__grow@15069316 ]
[dart     : sp(0x7f853cd7bce0) fp(0x7f853cd7bd28) pc(0x7f8510ecba58) dart:io__CopyingBytesBuilder@15069316_add ]
[dart     : sp(0x7f853cd7bd38) fp(0x7f853cd7bd88) pc(0x7f84fc732707) *package:kernel/binary/ast_to_binary.dart_BufferedSink_addBytes ]
[dart     : sp(0x7f853cd7bd98) fp(0x7f853cd7bdc8) pc(0x7f8510ecf50d) package:kernel/binary/ast_to_binary.dart_BinaryPrinter_writeBytes ]
[dart     : sp(0x7f853cd7bdd8) fp(0x7f853cd7be98) pc(0x7f850d209edd) *package:kernel/binary/ast_to_binary.dart_BinaryPrinter_writeStringTable ]
[dart     : sp(0x7f853cd7bea8) fp(0x7f853cd7bef8) pc(0x7f8510eadc2b) package:kernel/binary/ast_to_binary.dart_BinaryPrinter_writeComponentFile_<anonymous closure> ]
[dart     : sp(0x7f853cd7bf08) fp(0x7f853cd7bf88) pc(0x7f8533b2298a) dart:developer_Timeline_timeSync ]
[dart     : sp(0x7f853cd7bf98) fp(0x7f853cd7bfd0) pc(0x7f8510ead753) package:kernel/binary/ast_to_binary.dart_BinaryPrinter_writeComponentFile ]
[dart     : sp(0x7f853cd7bfe0) fp(0x7f853cd7c028) pc(0x7f8510ead262) package:front_end/src/fasta/incremental_serializer.dart_IncrementalSerializer_serialize ]
[dart     : sp(0x7f853cd7c038) fp(0x7f853cd7c110) pc(0x7f8514fc2a4a) *package:front_end/src/fasta/incremental_serializer.dart_IncrementalSerializer_writePackagesToSinkAndTrimComponent ]
[dart     : sp(0x7f853cd7c120) fp(0x7f853cd7c1b8) pc(0x7f8510e9cdbd) file:///b/s/w/ir/cache/builder/sdk/pkg/frontend_server/lib/frontend_server.dart_FrontendCompiler_writeDillFile__async_op ]
[dart     : sp(0x7f853cd7c1c8) fp(0x7f853cd7c1e0) pc(0x7f85126db002) *dart:async__AsyncAwaitCompleter@9048458_start ]
[dart     : sp(0x7f853cd7c1f0) fp(0x7f853cd7c288) pc(0x7f8510e9c298) file:///b/s/w/ir/cache/builder/sdk/pkg/frontend_server/lib/frontend_server.dart_FrontendCompiler_writeDillFile ]
[dart     : sp(0x7f853cd7c298) fp(0x7f853cd7c370) pc(0x7f8534235666) file:///b/s/w/ir/cache/builder/sdk/pkg/frontend_server/lib/frontend_server.dart_FrontendCompiler_compile__async_op ]
[dart     : sp(0x7f853cd7c380) fp(0x7f853cd7c3d0) pc(0x7f853425f829) dart:async__RootZone@9048458_runUnary ]
[dart     : sp(0x7f853cd7c3e0) fp(0x7f853cd7c440) pc(0x7f850fa24563) *dart:async__FutureListener@9048458_handleValue ]
[dart     : sp(0x7f853cd7c450) fp(0x7f853cd7c4d0) pc(0x7f850fa7bef2) *dart:async__Future@9048458__propagateToListeners@9048458_handleValueCallback ]
[dart     : sp(0x7f853cd7c4e0) fp(0x7f853cd7c538) pc(0x7f850fa1ca9d) *dart:async__Future@9048458__propagateToListeners@9048458 ]
[dart     : sp(0x7f853cd7c548) fp(0x7f853cd7c588) pc(0x7f8515f2a48d) *dart:async__Future@9048458__completeWithValue@9048458 ]
[dart     : sp(0x7f853cd7c598) fp(0x7f853cd7c5c8) pc(0x7f85256241e7) *dart:async_::__completeOnAsyncReturn@9048458 ]
[dart     : sp(0x7f853cd7c5d8) fp(0x7f853cd7c660) pc(0x7f8534274227) package:vm/incremental_compiler.dart_IncrementalCompiler_compile__async_op ]
[dart     : sp(0x7f853cd7c670) fp(0x7f853cd7c6a8) pc(0x7f84fc74e673) *dart:async_::__asyncThenWrapperHelper@9048458_<anonymous closure> ]
[dart     : sp(0x7f853cd7c6b8) fp(0x7f853cd7c738) pc(0x7f853bf4d3f8) *dart:async_::__rootRunUnary@9048458 ]
[dart     : sp(0x7f853cd7c748) fp(0x7f853cd7c7c8) pc(0x7f8515f3b187) *dart:async_::__rootRunUnary@9048458__rootRunUnary@9048458 ]
[dart     : sp(0x7f853cd7c7d8) fp(0x7f853cd7c840) pc(0x7f851823da32) *dart:async__CustomZone@9048458_runUnary ]
[dart     : sp(0x7f853cd7c850) fp(0x7f853cd7c8b0) pc(0x7f850fa24541) *dart:async__FutureListener@9048458_handleValue ]
[dart     : sp(0x7f853cd7c8c0) fp(0x7f853cd7c940) pc(0x7f850fa7bef2) *dart:async__Future@9048458__propagateToListeners@9048458_handleValueCallback ]
[dart     : sp(0x7f853cd7c950) fp(0x7f853cd7c9a8) pc(0x7f850fa1ca9d) *dart:async__Future@9048458__propagateToListeners@9048458 ]
[dart     : sp(0x7f853cd7c9b8) fp(0x7f853cd7c9f8) pc(0x7f8515f2a48d) *dart:async__Future@9048458__completeWithValue@9048458 ]
[dart     : sp(0x7f853cd7ca08) fp(0x7f853cd7ca38) pc(0x7f85256241e7) *dart:async_::__completeOnAsyncReturn@9048458 ]
[dart     : sp(0x7f853cd7ca48) fp(0x7f853cd7cf10) pc(0x7f850fa096ba) *package:front_end/src/fasta/incremental_compiler.dart_IncrementalCompiler_computeDelta__async_op_<anonymous closure>__async_op ]
[dart     : sp(0x7f853cd7cf20) fp(0x7f853cd7cf58) pc(0x7f84fc74e673) *dart:async_::__asyncThenWrapperHelper@9048458_<anonymous closure> ]
[dart     : sp(0x7f853cd7cf68) fp(0x7f853cd7cfe8) pc(0x7f853bf4d3f8) *dart:async_::__rootRunUnary@9048458 ]
[dart     : sp(0x7f853cd7cff8) fp(0x7f853cd7d078) pc(0x7f8515f3b187) *dart:async_::__rootRunUnary@9048458__rootRunUnary@9048458 ]
[dart     : sp(0x7f853cd7d088) fp(0x7f853cd7d0f0) pc(0x7f851823da32) *dart:async__CustomZone@9048458_runUnary ]
[dart     : sp(0x7f853cd7d100) fp(0x7f853cd7d160) pc(0x7f850fa24541) *dart:async__FutureListener@9048458_handleValue ]
[dart     : sp(0x7f853cd7d170) fp(0x7f853cd7d1f0) pc(0x7f850fa7bef2) *dart:async__Future@9048458__propagateToListeners@9048458_handleValueCallback ]
[dart     : sp(0x7f853cd7d200) fp(0x7f853cd7d258) pc(0x7f850fa1ca9d) *dart:async__Future@9048458__propagateToListeners@9048458 ]
[dart     : sp(0x7f853cd7d268) fp(0x7f853cd7d2a8) pc(0x7f8515f2a48d) *dart:async__Future@9048458__completeWithValue@9048458 ]
[dart     : sp(0x7f853cd7d2b8) fp(0x7f853cd7d2e8) pc(0x7f85256241e7) *dart:async_::__completeOnAsyncReturn@9048458 ]
[dart     : sp(0x7f853cd7d2f8) fp(0x7f853cd7d388) pc(0x7f8533654b10) package:front_end/src/fasta/crash.dart_::_withCrashReporting__async_op ]
[dart     : sp(0x7f853cd7d398) fp(0x7f853cd7d3d0) pc(0x7f84fc74e673) *dart:async_::__asyncThenWrapperHelper@9048458_<anonymous closure> ]
[dart     : sp(0x7f853cd7d3e0) fp(0x7f853cd7d460) pc(0x7f853bf4d3f8) *dart:async_::__rootRunUnary@9048458 ]
[dart     : sp(0x7f853cd7d470) fp(0x7f853cd7d4f0) pc(0x7f8515f3b187) *dart:async_::__rootRunUnary@9048458__rootRunUnary@9048458 ]
[dart     : sp(0x7f853cd7d500) fp(0x7f853cd7d568) pc(0x7f851823da32) *dart:async__CustomZone@9048458_runUnary ]
[dart     : sp(0x7f853cd7d578) fp(0x7f853cd7d5d8) pc(0x7f850fa24541) *dart:async__FutureListener@9048458_handleValue ]
[dart     : sp(0x7f853cd7d5e8) fp(0x7f853cd7d668) pc(0x7f850fa7bef2) *dart:async__Future@9048458__propagateToListeners@9048458_handleValueCallback ]
[dart     : sp(0x7f853cd7d678) fp(0x7f853cd7d6d0) pc(0x7f850fa1ca9d) *dart:async__Future@9048458__propagateToListeners@9048458 ]
[dart     : sp(0x7f853cd7d6e0) fp(0x7f853cd7d720) pc(0x7f8515f2a48d) *dart:async__Future@9048458__completeWithValue@9048458 ]
[dart     : sp(0x7f853cd7d730) fp(0x7f853cd7d760) pc(0x7f85256241e7) *dart:async_::__completeOnAsyncReturn@9048458 ]
[dart     : sp(0x7f853cd7d770) fp(0x7f853cd7d7f8) pc(0x7f851825d4fa) package:front_end/src/fasta/kernel/kernel_target.dart_KernelTarget_buildComponent__async_op_<anonymous closure>__async_op ]
[dart     : sp(0x7f853cd7d808) fp(0x7f853cd7d840) pc(0x7f84fc74e673) *dart:async_::__asyncThenWrapperHelper@9048458_<anonymous closure> ]
[dart     : sp(0x7f853cd7d850) fp(0x7f853cd7d8d0) pc(0x7f853bf4d3f8) *dart:async_::__rootRunUnary@9048458 ]
[dart     : sp(0x7f853cd7d8e0) fp(0x7f853cd7d960) pc(0x7f8515f3b187) *dart:async_::__rootRunUnary@9048458__rootRunUnary@9048458 ]
[dart     : sp(0x7f853cd7d970) fp(0x7f853cd7d9d8) pc(0x7f851823da32) *dart:async__CustomZone@9048458_runUnary ]
[dart     : sp(0x7f853cd7d9e8) fp(0x7f853cd7da48) pc(0x7f850fa24541) *dart:async__FutureListener@9048458_handleValue ]
[dart     : sp(0x7f853cd7da58) fp(0x7f853cd7dad8) pc(0x7f850fa7bef2) *dart:async__Future@9048458__propagateToListeners@9048458_handleValueCallback ]
[dart     : sp(0x7f853cd7dae8) fp(0x7f853cd7db40) pc(0x7f850fa1ca9d) *dart:async__Future@9048458__propagateToListeners@9048458 ]
[dart     : sp(0x7f853cd7db50) fp(0x7f853cd7db90) pc(0x7f8515f2a48d) *dart:async__Future@9048458__completeWithValue@9048458 ]
[dart     : sp(0x7f853cd7dba0) fp(0x7f853cd7dbd0) pc(0x7f85256241e7) *dart:async_::__completeOnAsyncReturn@9048458 ]
[dart     : sp(0x7f853cd7dbe0) fp(0x7f853cd7dd20) pc(0x7f8514fcaf46) *package:front_end/src/fasta/loader.dart_Loader_buildBodies__async_op ]
[dart     : sp(0x7f853cd7dd30) fp(0x7f853cd7dd68) pc(0x7f84fc74e673) *dart:async_::__asyncThenWrapperHelper@9048458_<anonymous closure> ]
[dart     : sp(0x7f853cd7dd78) fp(0x7f853cd7ddf8) pc(0x7f853bf4d3f8) *dart:async_::__rootRunUnary@9048458 ]
[dart     : sp(0x7f853cd7de08) fp(0x7f853cd7de88) pc(0x7f8515f3b187) *dart:async_::__rootRunUnary@9048458__rootRunUnary@9048458 ]
[dart     : sp(0x7f853cd7de98) fp(0x7f853cd7df00) pc(0x7f851823da32) *dart:async__CustomZone@9048458_runUnary ]
[dart     : sp(0x7f853cd7df10) fp(0x7f853cd7df70) pc(0x7f850fa24541) *dart:async__FutureListener@9048458_handleValue ]
[dart     : sp(0x7f853cd7df80) fp(0x7f853cd7e000) pc(0x7f850fa7bef2) *dart:async__Future@9048458__propagateToListeners@9048458_handleValueCallback ]
[dart     : sp(0x7f853cd7e010) fp(0x7f853cd7e068) pc(0x7f850fa1ca9d) *dart:async__Future@9048458__propagateToListeners@9048458 ]
[dart     : sp(0x7f853cd7e078) fp(0x7f853cd7e0b8) pc(0x7f8515f2a48d) *dart:async__Future@9048458__completeWithValue@9048458 ]
[dart     : sp(0x7f853cd7e0c8) fp(0x7f853cd7e0f8) pc(0x7f85256241e7) *dart:async_::__completeOnAsyncReturn@9048458 ]
[dart     : sp(0x7f853cd7e108) fp(0x7f853cd7e2b8) pc(0x7f84fc71b0f8) *package:front_end/src/fasta/source/source_loader.dart_SourceLoader_buildBody__async_op ]
[dart     : sp(0x7f853cd7e2c8) fp(0x7f853cd7e300) pc(0x7f84fc74e673) *dart:async_::__asyncThenWrapperHelper@9048458_<anonymous closure> ]
[dart     : sp(0x7f853cd7e310) fp(0x7f853cd7e390) pc(0x7f853bf4d3f8) *dart:async_::__rootRunUnary@9048458 ]
[dart     : sp(0x7f853cd7e3a0) fp(0x7f853cd7e420) pc(0x7f8515f3b187) *dart:async_::__rootRunUnary@9048458__rootRunUnary@9048458 ]
[dart     : sp(0x7f853cd7e430) fp(0x7f853cd7e498) pc(0x7f851823da32) *dart:async__CustomZone@9048458_runUnary ]
[dart     : sp(0x7f853cd7e4a8) fp(0x7f853cd7e508) pc(0x7f850fa24541) *dart:async__FutureListener@9048458_handleValue ]
[dart     : sp(0x7f853cd7e518) fp(0x7f853cd7e598) pc(0x7f850fa7bef2) *dart:async__Future@9048458__propagateToListeners@9048458_handleValueCallback ]
[dart     : sp(0x7f853cd7e5a8) fp(0x7f853cd7e600) pc(0x7f850fa1ca9d) *dart:async__Future@9048458__propagateToListeners@9048458 ]
[dart     : sp(0x7f853cd7e610) fp(0x7f853cd7e650) pc(0x7f8515f2a48d) *dart:async__Future@9048458__completeWithValue@9048458 ]
[dart     : sp(0x7f853cd7e660) fp(0x7f853cd7e6a0) pc(0x7f853425c81a) dart:async__Future@9048458__asyncComplete@9048458_<anonymous closure> ]
[dart     : sp(0x7f853cd7e6b0) fp(0x7f853cd7e738) pc(0x7f850fa65b69) *dart:async_::__rootRun@9048458 ]
[dart     : sp(0x7f853cd7e748) fp(0x7f853cd7e7b0) pc(0x7f84fc771f1b) *dart:async_::__rootRun@9048458__rootRun@9048458 ]
[dart     : sp(0x7f853cd7e7c0) fp(0x7f853cd7e820) pc(0x7f8514ffabaf) *dart:async__CustomZone@9048458_run ]
[dart     : sp(0x7f853cd7e830) fp(0x7f853cd7e860) pc(0x7f84fc7716f4) *dart:async__CustomZone@9048458_bindCallback_<anonymous closure> ]
[dart     : sp(0x7f853cd7e870) fp(0x7f853cd7e890) pc(0x7f84fc7351ce) *dart:async_::__microtaskLoop@9048458 ]
[dart     : sp(0x7f853cd7e8a0) fp(0x7f853cd7e8d0) pc(0x7f8534226211) dart:async_::__startMicrotaskLoop@9048458 ]
[dart     : sp(0x7f853cd7e8e0) fp(0x7f853cd7e910) pc(0x7f8534226127) dart:async_::__startMicrotaskLoop@9048458__startMicrotaskLoop@9048458 ]
[dart     : sp(0x7f853cd7e920) fp(0x7f853cd7e950) pc(0x7f8534225f26) dart:isolate_::__runPendingImmediateCallback@1026248 ]
[dart     : sp(0x7f853cd7e960) fp(0x7f853cd7e978) pc(0x7f853bf2afdd) dart:isolate__RawReceivePortImpl@1026248__handleMessage@1026248 ]
[entry    : sp(0x7f853cd7e988) fp(0x7f853cd7e9e8) pc(0x7f853cd8149c)]
```

/cc @mkustermann

**Comment by mkustermann** [comment]
This could be due the recent GC changes. One of the CLs got reverted already.

/cc @a-siva @rmacnak-google

**Comment by rmacnak-google** [comment]
This is the same crash as b/147582727. It is fixed by the revert in 67f3cf325d4eda3f73e27664f0254a95913ddc8e.
